docs/process_joined_data.js

```diff
@@ -22,8 +22,9 @@ const scoreDisplayCap = 255;
  */
 function colorAndRate(
     feature, scalingFactor, povertyThreshold, damageThreshold, povertyWeight) {
-  const povertyRatio = feature.properties[snapPercentageTag];
-  const ratioBuildingsDamaged = feature.properties[damageTag];
+  const povertyRatio = feature.properties[povertyPercentageTag];
+  // If damage tag is absent, treat as 0.
+  const ratioBuildingsDamaged = feature.properties[damageTag] || 0;
```

Comment:
Was this already happening but just more safe now?

Reply:
Nope, it would have been `undefined` and then the calculations below would have been bogus.